web/controllers/statistics_controller.ex

```diff
@@ -1,14 +1,12 @@
 defmodule Urito.StatisticsController do
   use Urito.Web, :controller
   alias Urito.MappedUrl
-  alias Urito.Statistics
 
   def index(conn, %{"mapped_url_id" => mapped_url_id}) do
-    statistics = MappedUrl
+    mapped_url = MappedUrl
                  |> Repo.get!(mapped_url_id)
-                 |> Repo.preload(:requests)
-                 |> Statistics.build
+                 |> Repo.preload(:statistics)
```

Comment:
Vim is lining these up for me when I reformat my file.

I don't mind it, and I don't feel like fighting whichever Vim plugin is responsible.

I'll stick with whatever gets formatted for now while I learn 🤔 .